docs/auth/google/gcp-iap-auth.md

```diff
@@ -77,6 +77,6 @@ backend.add(import('@backstage/plugin-auth-backend-module-gcp-iap-provider'));
 
 ## Adding the provider to the Backstage frontend
 
-See [Sign-In with Proxy Providers](../index.md#sign-in-with-proxy-providers) for pointers on how to set up the sign-in page, and to also make it work smoothly for local development. You'll use `gcp-iap` as the provider name.
+See [Sign-In with Proxy Providers](../index.md#sign-in-with-proxy-providers) for pointers on how to set up the sign-in page, and to also make it work smoothly for local development. You'll use `gcpiap` as the provider name.
```

Comment:
These are case sensitive aren't they - should this be `gcpIap`?

Reply:
I used `gcpiap` in my local configuration, and it worked, so I didn't really look any deeper. Looking at `auth-backend/src/providers/router.ts` and `auth-backend/src/service/router.ts`, it's using the default options in `express.Router`, which makes it case-insensitive. I can try it with `gcpIap` to make sure it works. If it does, then I'll update both these docs to make everything consistent.

Reply:
It was case-insensitive. Using `gcpIap` worked. I've updated this to `gcpIap` instead of `gcpiap`.